config/setup/xutil.adb

```diff
@@ -6,7 +6,7 @@
 --                                                                          --
 --                                 B o d y                                  --
 --                                                                          --
---          Copyright (C) 1992-2012, Free Software Foundation, Inc.         --
+--          Copyright (C) 1992-2018, Free Software Foundation, Inc.         --
```

Comment:
Why? There is no change in this file.

Reply:
I just copied it from gnat sources

Reply:
And I think it is a bit better to update then to do not update.